.github/workflows/devtools.yml

```diff
@@ -44,6 +33,17 @@ jobs:
       working-directory: ${{ github.workspace }}/lighthouse
     - run: yarn build-devtools
       working-directory: ${{ github.workspace }}/lighthouse
+    
+    # Caches last ~week, so CI will refresh these dependencies that often.
+    - name: Cache depot tools, devtools, blink tools and content shell
+      uses: actions/cache@v2
+      with:
+        path: |
+          ${{ env.DEPOT_TOOLS_PATH }}
+          ${{ env.DEVTOOLS_PATH }}
+          ${{ env.BLINK_TOOLS_PATH }}
+          ${{ github.workspace }}/lighthouse/.tmp/chromium-web-tests/content-shells
+        key: ${{ runner.os }}-${{ hashFiles('lighthouse/.github/workflows/devtools.yml', 'lighthouse/dist/lighthouse-dt-bundle.js', 'lighthouse/lighthouse-core/test/chromium-web-tests/download-*') }}
```

Comment:
> `lighthouse/dist/lighthouse-dt-bundle.js`

won't this mean it won't use the cache on basically any change to core?

Reply:
shoot, good point. I will use the build bundle, devtools entry, and devtools reports files instead.